amethyst_renderer_new/src/back.rs

```diff
@@ -0,0 +1,39 @@
+
+#[cfg(feature = "metal")]
```

Comment:
Shouldn't we at some point allow to dynamically choose the backend? I think for an actual game it's preferable to have one binary which supports all the backends.

Reply:
I'll think about it